openapi/sms/v1-review/docs/readme.md

```diff
@@ -0,0 +1,262 @@
+---
+stoplight-id: cz3ydei53qosx
+---
+
+# Emarsys SMS Partner API Documentation
+
+SAP Emarsys empowers marketers around the world to create a truly personalized omnichannel experiences that delivers business outcomes. The Emarsys SMS Partner API allows partners to integrate onto the Emarsys SMS channel. The Partner API uses a RESTful approach.
+
+## API Specification
+
+The API specification has been created using OpenAPI 3.0.3. It is available to be viewed and exported using the links below:
+
+- [Partner Service Specification](../partner-service.yaml)
+- [Partner Callbacks Specification](../partner-callbacks.yaml)
```

Comment:
in stoplight, it is called Provider Callbacks. 
![Screenshot 2023-04-03 at 13 22 40](https://user-images.githubusercontent.com/313458/229495401-70d3f9aa-fa93-47d2-86b9-d23602162bb3.png)

Furthermore, could we describe the purpose of those APIs shortly?

Reply:
updating the title, the description in the YAML and also adding `Partner Callbacks` definition to the glossary. thanks!